From f38ea1f7596f7021c9b3df25df800f141ef6c185 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Sun, 20 Oct 2024 17:02:41 +0200
Subject: [PATCH] Revert background color

---
 pyironflow/reactflow.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyironflow/reactflow.py b/pyironflow/reactflow.py
index ac3bec0..f067e54 100644
--- a/pyironflow/reactflow.py
+++ b/pyironflow/reactflow.py
@@ -145,7 +145,7 @@ def get_node_dict(node, id_num, key=None):
         'type': 'customNode',
         'style': {'border': '1px black solid',
                   'padding': 5,
-                  'background': node.gui_color,  # '#1999',
+                  'background': '#1999', # node.gui_color,  # This requires Tara's branch of pyiron_workflow
                   'borderRadius': '10px',
                   'width': f'{node_width}px'},
         'targetPosition': 'left',
