From ed2e831cbfe4f12aba99d4471b97b16db9c6058b Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Sun, 20 Oct 2024 10:01:58 +0200
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 634e50e..a1624d6 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -39,15 +39,12 @@ Repository = "https://github.com/pyiron/pyironFlow"
 [project.optional-dependencies]
 dev = ["watchfiles", "jupyterlab"]
 
-[tool.setuptools.packages.find]
-include = ["pyironflow*"]
-
 [tool.hatch.envs.default]
 features = ["dev"]
 
 [tool.hatch.build]
 only-packages = true
-artifacts = ["pyironflow/static/*"]
+artifacts = ["pyironflow/*", "pyironflow/static/*"]
 
 [tool.hatch.build.hooks.jupyter-builder]
 build-function = "hatch_jupyter_builder.npm_builder"
