From 2b46bb0cb3ea013ac1f52702c9c2db0d7ffa90c7 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Sun, 20 Oct 2024 08:08:31 +0200
Subject: [PATCH] root_path - make root_path available in PyironFlow

---
 pyironflow/pyironflow.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyironflow/pyironflow.py b/pyironflow/pyironflow.py
index 37cda93..07d27b5 100644
--- a/pyironflow/pyironflow.py
+++ b/pyironflow/pyironflow.py
@@ -17,7 +17,7 @@
 
 
 class PyironFlow:
-    def __init__(self, wf_list=None):
+    def __init__(self, wf_list=None, root_path='../pyiron_nodes/pyiron_nodes'):
         # generate empty default workflow if workflow list is empty
         if wf_list is None:
             wf_list = []
@@ -31,7 +31,7 @@ def __init__(self, wf_list=None):
         self.wf_widgets = [PyironFlowWidget(wf, log=self.out_log, out_widget=self.out_widget)
                            for wf in self.workflows]
         self.view_flows = self.view_flows()
-        self.tree_view = TreeView(flow_widget=self.wf_widgets[0], log=self.out_log)
+        self.tree_view = TreeView(root_path=root_path, flow_widget=self.wf_widgets[0], log=self.out_log)
         self.accordion = widgets.Accordion(children=[self.tree_view.gui, self.out_widget, self.out_log],
                                            titles=['Node Library', 'Output', 'Logging Info'],
                                            layout={'border': '1px solid black', 'min_width': '400px'})
